concent_api/core/message_handlers.py

```diff
@@ -438,37 +439,34 @@ def handle_send_force_subtask_results(client_message: message.concents.ForceSubt
     current_time = get_current_utc_timestamp()
 
     if Subtask.objects.filter(
-        subtask_id = task_to_compute.compute_task_def['subtask_id'],
-        state      = Subtask.SubtaskState.FORCING_ACCEPTANCE.name,  # pylint: disable=no-member
+        subtask_id=task_to_compute.compute_task_def['subtask_id'],
+        state=Subtask.SubtaskState.FORCING_ACCEPTANCE.name,  # pylint: disable=no-member
     ).exists():
         return message.concents.ServiceRefused(
-            reason = message.concents.ServiceRefused.REASON.DuplicateRequest,
+            reason=message.concents.ServiceRefused.REASON.DuplicateRequest,
         )
 
     if not core.payments.base.is_account_status_positive(  # pylint: disable=no-value-for-parameter
-        client_eth_address      = client_message.ack_report_computed_task.report_computed_task.task_to_compute.requestor_ethereum_address,
-        pending_value           = client_message.ack_report_computed_task.report_computed_task.task_to_compute.price,
+        client_eth_address=task_to_compute.requestor_ethereum_address,
+        pending_value=task_to_compute.price,
     ):
         return message.concents.ServiceRefused(
-            reason      = message.concents.ServiceRefused.REASON.TooSmallRequestorDeposit,
+            reason=message.concents.ServiceRefused.REASON.TooSmallRequestorDeposit,
         )
 
     core.payments.base.make_force_payment_to_provider(  # pylint: disable=no-value-for-parameter
-        requestor_eth_address = client_message.ack_report_computed_task.report_computed_task.task_to_compute.requestor_ethereum_address,
-        provider_eth_address = client_message.ack_report_computed_task.report_computed_task.task_to_compute.provider_ethereum_address,
-        value = client_message.ack_report_computed_task.report_computed_task.task_to_compute.price,
-        payment_ts = current_time,
+        requestor_eth_address=task_to_compute.requestor_ethereum_address,
+        provider_eth_address=task_to_compute.provider_ethereum_address,
+        value=task_to_compute.price,
+        payment_ts=current_time,
     )
 
-    verification_deadline       = (
-        int(client_message.ack_report_computed_task.report_computed_task.task_to_compute.compute_task_def['deadline']) +
-        calculate_subtask_verification_time(client_message.ack_report_computed_task.report_computed_task)
-    )
-    forcing_acceptance_deadline = (
-        int(client_message.ack_report_computed_task.report_computed_task.task_to_compute.compute_task_def['deadline']) +
-        calculate_subtask_verification_time(client_message.ack_report_computed_task.report_computed_task) +
-        settings.FORCE_ACCEPTANCE_TIME
-    )
+    task_deadline = int(task_to_compute.compute_task_def['deadline'])
+    subtask_verification_time = calculate_subtask_verification_time()
```

Comment:
I'm getting `No value for argument 'report_computed_task' in function call (no-value-for-parameter)` from pylint, which make sense.

Reply:
:+1: done